systems/registry/docker-compose.yml

```diff
@@ -160,7 +160,8 @@ services:
    - postgresd-registry
 
  initclient-registry:
-  build: ../services/initClient
+  # build: ../services/initClient
+  image: main-init
```

Comment:
Are we pushing main-init initclient image config in main? I don't think weshould.

Reply:
Thanks for catching it 👍